ulx/src/main/java/com/uplink/ulx/drivers/bluetooth/ble/controller/BleBrowser.java

```diff
@@ -768,12 +686,12 @@ public void onConnected(@NonNull Connector connector) {
         );
 
         Dispatch.post(() -> {
+            connectionsInProgress--;
+            if (connectionsInProgress == 0) { // We went from 1 to 0 - maybe time to stop scanning
```

Comment:
here you mean start scanning

Reply:
@therock24 Yes, if we are currently requested to scan (via `requestAdapterToStartBrowsing()`)